imap_processing/idex/idex_l2a.py

```diff
@@ -537,8 +537,19 @@ def estimate_dust_mass(
     #         information soon.
     signal = np.array(target_signal.data)
     time = np.array(low_sampling_time.data)
+    mask = np.logical_and(
```

Comment:
I generally think of mask as "bad" values. I'd suggest calling this `good_mask`, `good_vals`, something like that or invert the logic to make the mask the bad values for consistency across the project.